lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -129,7 +137,7 @@ def _call(cmd: str, *args: str, stdin: Optional[str] = None) -> str:
     """Call a command with logging.
 
     Raises:
-        subprocess.CalledProcessError: Raised if the command fails.
+        SlurmManagerError: Raised if the command fails.
```

Comment:
I think you mean `SlurmOpsError` :smile: 

```suggestion
        SlurmOpsError: Raised if the command fails.
```

Reply:
Oops